data.sql

```diff
@@ -1,6 +1,9 @@
 /* Populate database with sample data. */
 
-insert into animals values (1,'Agumon','2020-02-03',0,false,10.23);
-insert into animals values (2,'Gabumon','2018-11-15',2,true,8);
-insert into animals values (3,'Pikachu','2021-01-07',1,true,15.04);
-insert into animals values (4,'Devimon','2017-05-12',5,true,11);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (5,'Charmander','2020-02-08',0,false,-11);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (6,'Plantmon','2021-11-15',2,true,-5.7);
+insert into animals(id,name,date_of_birth,escape_attempts,neutered,weigth_kg) values (7,'Squirtle','1993-03-02',3,false,-12.13);
```

Comment:
* Kindly add the correct date of birth for the animal named `Squirtle` DOB: `He was born on Apr 2nd, 1993,` [According to the requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=He%20was%20born%20on%20Apr%202nd%2C%201993%2C)